src/lib/core/datetime/native-date-adapter.ts

```diff
@@ -300,4 +300,21 @@ export class NativeDateAdapter extends DateAdapter<Date> {
   private _stripDirectionalityCharacters(str: string) {
     return str.replace(/[\u200e\u200f]/g, '');
   }
+
+  /**
+   * When converting Date object to string, javascript built-in functions may return wrong
+   * results because it applies its internal DST rules. The DST rules around the world change
+   * very frequently, and the current valid rule is not always valid in previous years though.
+   * We work around this problem building a new Date object which has its internal UTC
+   * representation with the local date and time.
+   * @param dtf Intl.DateTimeFormat object, containg the desired string format
```

Comment:
Add a note that `dtf` must have its `timeZone` set to `'utc'` for this method to work properly